ivy/stateful/layers.py

```diff
@@ -2224,3 +2224,58 @@ def _forward(self, x):
             The input array as it is.
         """
         return x
+
+
+class IFFT(Module):
+    def __init__(
+        self,
+        dim,
+        /,
+        *,
+        norm="backward",
+        n=None,
+        out=None,
+        device=None,
+        dtype=None,
+    ):
+        """
+        Class for applying IFFT to input.
+
+        Parameters
+        ----------
+        dim : int
+            Dimension along which to take the IFFT.
+        norm : str
+            Normalization mode. Default: 'backward'
+        n : int
+            Size of the IFFT. Default: None
+        out : int
+            Size of the output. Default: None
+        """
+        self._dim = dim
+        self._norm = norm
+        self._n = n
+        self._out = out
+        Module.__init__(self, device=device, dtype=dtype)
+
+    def _forward(self, inputs):
+        """
+        Forward pass of the layer.
+
+        Parameters
+        ----------
+        inputs : array
+            Input array to take the IFFT of.
+
+        Returns
+        -------
+        array
```

Comment:
There is no variable named `array` that is returned, so you could remove this line.

Reply:
thanks @bipinKrishnan ..have made the changes